### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by epicmiller**
Added required `new` keyword when calling compileSass and documented the `sassDir` option.

### CODE REVIEW

**Comment by epicmiller** [comment]
I also wanted to ask, is it okay that we need to specify `new` before calling the eyeglass compiler in Broccoli? From what I've seen its more Broccoli convention to export a plain old function, not a constructor. If you don't specify `new` it throws a very hard to track down error. This can be fixed by adding an `EyeglassCompilerFactory` to `index.js`:

``` javascript
function EyeglassCompilerFactory(tree, options){
    return new EyeglassCompiler(tree, options);
}

module.exports = EyeglassCompilerFactory;
```

**Comment by eoneill** [comment]
Could also do something like...

```js
function EyeglassCompiler(tree, options) {
  // if it's not already an instance, return a new instance
  if (!(this instanceof EyeglassCompiler)) {
    return new EyeglassCompiler(tree, options);
  }

  ...
}
```

so `EyeglassCompiler` and `new EyeglassCompiler` can be used interchangeably.

**Comment by epicmiller** [comment]
`EyeglassCompiler` extends `BroccoliSassCompiler` which ultimately inherits from a package called `core-object`: https://github.com/ember-cli/core-object

Would have to be something like:

``` javascript
function EyeglassCompiler(tree, options) {
  // if it's not already an instance, return a new instance
  if (!(this instanceof EyeglassCompiler)) {
    return new EyeglassCompiler(tree, options);
  }
}

EyeglassCompiler.__proto__ = BroccoliSassCompiler;
EyeglassCompiler.prototype =  Object.create(BroccoliSassCompiler.prototype);

EyeglassCompiler = EyeglassCompiler.extend({
  init: function(inputTrees, options) {
    ...
  },

  handleNewFile: function(details) {
    ...
  },

  updateCache: function(srcPaths, destDir) {
    ...
  }
});

module.exports = EyeglassCompiler;
```

The factory may be simpler :)

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ var options = {
   cssDir: "assets/css",
   fullException: false
 }
-var outputTree = compileSass(inputTrees, options);
+var outputTree = new compileSass(inputTrees, options);
```
Comment (alanhogan) [inline README.md]: @eoneill This looks funny (class names should be capitalized.) I’d like to see the `new` keyword be optional, as discussed in this PR’s comments

**Pull request merged**